code-challenges.js

```diff
@@ -23,19 +53,74 @@ var middleLetters1 = "hello"
 var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
 
+const middleChar = (string) => {
+    //Uses a ternary operator to check if a string has an even or odd amount of letters. If the string has an odd amount of letters, it will return the letter that is at the in the middle by dividing the string in half and rounding that number up. If the string has an even amount of letters, it will return the middle two letters by substringing the first middle letter (inclusive) and the next two letters (exclusive).
+    return string.length % 2 === 1 ? string[Math.ceil(string.length / 2)] : string.substring(string.length / 2 - 1, string.length / 2 + 1)
+
+}
```

Comment:
I love how much you pseudocode an explain your solution. thats a great habit to have!